src/NuGetPackageVerifier/Rules/AssemblyHasAttributeHelper.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Mono.Cecil;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class AssemblyHasAttributeHelper
+    {
+        public static void GetAssemblyAttributesData(PackageAnalysisContext context)
```

Comment:
`GetXyz` methods sound like they return some data, but this one doesn't. Can we come up with a better method name for this?